stripe/_stripe_client.py

```diff
@@ -146,7 +152,7 @@ def __init__(
             **base_addresses,
         }
 
-        requestor_options = RequestorOptions(
+        self._requestor_options = RequestorOptions(
```

Comment:
What was the reason for this change?

Reply:
That was so I could reference those api_key, stripe_version, and stripe_account down in deserialize without having to add several additional member variables.

Reply:
Where do we use this in deserialize method?